modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalPreviewArticleContentTemplateDisplayContext.java

```diff
@@ -104,6 +106,23 @@ public long getDDMTemplateId() {
 		return _ddmTemplateId;
 	}
 
+	public String getDDMTemplateJSON() {
+		if ((getDDMTemplateId() == -1) || (getDDMTemplateId() == 0)) {
```

Comment:
Hi @jkappler, 
I made it this way because -1 and 0 have different meanings. -1 represents the value when we want to clear the selection. 0 is the initial value when there is no selection. If it is better in a simplified version I can make the changes.

Reply:
Hi @bakayattila, I understand, but with a simplified version we can also control if ddmTemplateId param is passed incorrectly

Reply:
I changed it as requested and added a playwright test. I simplified the steps to make it easer to test.